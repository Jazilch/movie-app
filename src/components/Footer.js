'use es6';

import React from 'react';
import styled from 'styled-components/macro';

const FooterWrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
`;

const Footer = () => (
  <FooterWrapper>
    <p>
      Created with{' '}
      <span aria-label="heart" role="img">
        ❤️
      </span>{' '}
      by James Zilch
    </p>
  </FooterWrapper>
);

export default Footer;
