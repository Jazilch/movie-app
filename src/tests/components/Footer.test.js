/* eslint-disable no-undef */

'use es6';

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../components/Footer';

describe('Footer component', () => {
  let footer;

  beforeEach(() => {
    footer = shallow(<Footer />);
  });

  it('renders component correctly', () => {
    expect(toJson(footer)).toMatchSnapshot();
  });

  it('displays the correct copyright info', () => {
    expect(footer.find('p').text()).toEqual('Created with ❤️ by James Zilch');
  });
});
