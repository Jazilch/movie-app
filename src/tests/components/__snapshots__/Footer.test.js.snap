// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders component correctly 1`] = `
<styled.div>
  <p>
    Created with
     
    <span
      aria-label="heart"
      role="img"
    >
      ❤️
    </span>
     
    by James Zilch
  </p>
</styled.div>
`;
